js/build.js

```diff
@@ -41,7 +41,7 @@
       function refreshData() {
         if (typeof data.dataSourceQuery !== 'object') {
           data.entries = [
-            { name: 'A', y: 3, sliced: true, selected: true },
+            { name: 'A', y: 3, sliced: false, selected: false },
```

Comment:
Do we need to set them as `false`, or can we just avoid setting any values? The default values might be `false`.

```suggestion
            { name: 'A', y: 3 },
```

Reply:
Based on https://api.highcharts.com/highcharts/series.pie.data.selected I think we can just leave out the property assignment because one defaults to `false` and another to `undefined`.

Reply:
I tested without assignment the property , it works as expected.

Reply:
Code review suggestion accepted and fixed.